homeassistant/components/zwave.py

```diff
@@ -417,11 +419,11 @@ def test_network(service):
         """Test the network by sending commands to all the nodes."""
         NETWORK.test()
 
-    def stop_zwave(event):
+    def stop_zwave(service):
```

Comment:
or actually, call it `_service_or_event` to be clear the method is used for both.